app/helpers/datasette_helper.rb

```diff
@@ -0,0 +1,32 @@
+# Helpers for Datasette integration
+module DatasetteHelper
+  include InfoRequestHelper
+
+  mattr_accessor :datasette_url, default: 'https://lite.datasette.io/'
+
+  def explore_in_datasette(attachment)
+    return unless explorable_in_datasette?(attachment)
+
+    link_to "#{datasette_url}?csv=#{attachment_url(attachment)}" do
+      _('Explore in Datasette')
+    end
```

Comment:
Could break depending on what `datasette_url` is set to, with #8030 we could do:

```suggestion
    datasette_attachment_url = add_query_params_to_url(
      datasette_url, csv: attachment_url(attachment)
    )

    link_to _('Explore in Datasette'), datasette_attachment_url
```

Reply:
Nice will incorporate https://github.com/mysociety/alaveteli/pull/8030 once merged

Reply:
Merged